src/main/webapp/WEB-INF/nav.jsp

```diff
@@ -6,31 +6,27 @@
   To change this template use File | Settings | File Templates.
 --%>
 <%@ page contentType="text/html;charset=UTF-8" language="java" %>
-<%-- navbar --%>
-<nav class="navbar navbar-expand-lg mb-5 bg-white rounded">
-  <%-- container --%>
+<nav class="navbar navbar-expand-lg mb-5 bg-white rounded shadow">
   <div class="container-fluid">
     <%-- logo --%>
     <img src="${pageContext.request.contextPath}/images/pixel-plant.png" class="plant-logo-sizing">
-    <%-- app name --%>
-    <a class="navbar-brand fw-bold" href="#">plantlookerupper &#8226</a>
-    <%-- toggler (smaller displays) --%>
-    <button class="navbar-toggler" type="button" data-bs-toggle="collapse" data-bs-target="#navbarTogglerDemo02" aria-controls="navbarTogglerDemo02" aria-expanded="false" aria-label="Toggle navigation">
+    <%-- project name --%>
+    <a class="navbar-brand fw-bold" href="${pageContext.request.contextPath}/">plantlookerupper</a>
+    <button class="navbar-toggler" type="button" data-bs-toggle="collapse" data-bs-target="#navbarToggler" aria-controls="navbarToggler" aria-expanded="false" aria-label="Toggle navigation">
       <span class="navbar-toggler-icon"></span>
     </button>
-    <div class="collapse navbar-collapse">
-      <%-- navbar list --%>
+    <div class="collapse navbar-collapse" id="navbarToggler">
       <ul class="navbar-nav me-auto mb-2 mb-lg-0">
         <li class="nav-item">
-          <a class="nav-link active" aria-current="page" href="#">Home &#8226</a>
+          <a class="nav-link" href="${pageContext.request.contextPath}/">Home</a>
         </li>
         <li class="nav-item">
-          <a class="nav-link" href="#">Search &#8226</a>
+          <a class="nav-link" href="${pageContext.request.contextPath}/search">Search</a>
```

Comment:
We may not need this link as we could just use the RESTful API I've created to output JSON data to the page. Let me know if this is not the case.